app/views/recipes/general_shopping_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<h3>Shopping list</h3>
+<div class="amount">
+  <p>Amount of food to buy: <%= @needed_items.length %></p>
+</div>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Price</th>
+    </tr>
+  </thead>
+  <% @needed_items.each do |needed_item| %>
+    <tbody>
+      <tr>
+        <td><%= needed_item[0] %></td>
+        <td><%= needed_item[1]  %><%= needed_item[3] %></td>
+        <td><%= needed_item[1] * needed_item[2] %></td>
+      </tr>
+    </tbody>
+  <% end %>
+</table>
```

Comment:
- [ ] It would be nice if you could add padding to the layout of the shopping list table so things do not show end2end of the screen. This will help make UI friendly 👍🏽
- [ ] From the wireframe, it's expected to display the value of the total food needed. Kindly display that on your UI screen just as you've displayed the count of foods 👍🏽

**Your UI**

<img width="1733" alt="Screenshot 2023-10-20 at 12 46 15 PM" src="https://github.com/tajulafreen/Recipe-app/assets/50941074/24853dd2-0102-49d7-a57a-b0b4bbd4d753">

**Wireframe UI**
<img width="1226" alt="Screenshot 2023-10-20 at 1 20 50 PM" src="https://github.com/tajulafreen/Recipe-app/assets/50941074/9ae9c618-3055-4c94-bb38-5af8905847cf">